reporters/color_reporter.py

```diff
@@ -0,0 +1,31 @@
+from pylint.reporters import BaseReporter
+
+import sys
+from colorama import Fore, Style
+from colorama import init
+
+
+class ColorReporter(BaseReporter):
+    def __init__(self):
+        super().__init__(self)
+        self._messages = []
+
+    def handle_message(self, msg):
+        """Handle a new message triggered on the current file."""
+        self._messages.append(msg)
+
+    def print_message_ids(self):
+        # Sort the messages by their type.
+        self._messages.sort(key=lambda s: s[0])
+        # Check if the OS currently running is Windows
+
+        init(wrap=(sys.platform == "win32"))
+        for msg in self._messages:
+            if msg.msg_id.startswith('E'):
+                # Error codes appear in red
+                # code = '\033[1;31m' + msg.msg_id + '\033[0m:'
```

Comment:
You can delete this now (and the comment below). Our goal is to hide what the ANSI codes actually are.